fixtures/allow_additional_props.json

```diff
@@ -192,9 +188,7 @@
           },
           "type": "array"
         },
-        "raw": {
-          "additionalProperties": true
-        }
+        "raw": {}
```

Comment:
`additionalProperties` should be used only with `"type": "object"`, but here we in fact want to allow any JSON fragment, so `{}` is the right schema for that.

Reply:
I've been working on our fork of jsonschema and added explicit support for this. The "idiomatic" approach for this case is to simply add `true` instead of `{}`. You can see the work in progress here: https://github.com/invopop/jsonschema/pull/3

Reply:
This PR I think makes this look better: https://github.com/invopop/jsonschema/pull/4